### DISCUSSION TITLE
极客蓝

### DISCUSSION THREAD

**Discussion by wuwenjun9939** [Meloong-Git/PCL#3694] [讨论]
你们是怎么解锁极客蓝的，没啥，就问问（别问为啥我有pre，问就是别人给的版本....）
应该不会把那人的内测和我的删了吧

**Comment by allMagicNB** [comment]
#3386

**Comment by allMagicNB** [comment]
![image](https://github.com/Hex-Dragon/PCL2/assets/116283787/ef574a8b-590d-4db1-8587-f5be6b4c66e7)

**Comment by WForst-Breeze** [comment]
快照版**严禁进行二次分发**，非法通过不正规渠道获取快照版是**不尊重软件开发者的表现**，请务必不要使用他人私自分发的快照版本 PCL。

同时，快照版不稳定，且如若没有更新密钥则无法获取稳定的更新，会对使用造成一定程度的困扰。

此外，对于非法取得的 PCL 程序，所造成的一切问题的后果**由使用者自行承担**，PCL 开发者对此类问题**概不负责**。

**Reply by wuwenjun9939** [reply]
更新密钥就不用纠结了/ww（手动狗头），别忘了，我开头是说了，朋友给的，当然也会有更新渠道啊
再说，所有主题的解锁手段被朋友从B站扒出来了（具体链接：https://www.bilibili.com/read/cv12788169/）
还有，快照版我看了一下，你确实开放了可以分享的
![image](https://github.com/Hex-Dragon/PCL2/assets/110022109/ee2bc05a-81b5-4bd8-b4cf-bd2d1bc76f88)

**Reply by MoYuan-CN** [reply]
通过朋友赞助 ￥54.32 与分享获得的快照版没有问题
我们的社区协作者朋友怕是通过其他渠道得来的所以发了这段话
你的快照版没有问题哈

**Reply by wuwenjun9939** [reply]
要没问题那就没啥事了，还以为从别人那获取的pcl是盗版（也可能叫侵权），然后朋友和我一起被龙猫起S就好玩了（doge）

**Reply by wuliaodexiaoluo** [reply]
没有盗版，只有侵权
因为快照版没有源码所以也不太可能有盗版

**Comment by wuliaodexiaoluo** [comment]


**Reply by LTCatt** [reply]
我多久默许 23.33 档可以分享快照版了？你是不是有病

**Reply by wuliaodexiaoluo** [reply]
记错了抱歉（
~这波直接把龙猫得罪死了~

**Comment by iPodtouch6557** [comment]
娃去这里敢说这个啊🤯💽📊
所以我觉得要不要对这个discu处置一下

**Reply by Deep-Dark-Forest** [reply]
没必要的，大家不看这个 discussion 都明白二次分发是怎么回事
毕竟原本付费的东西你给免费拿到了那谁都明白是怎么回事了

**Reply by WForst-Breeze** [reply]
没法溯源吧（摊手）

**Reply by MoYuan-CN** [reply]
我记得龙猫在哪里说过有溯源手段，不公开的

**Reply by Deep-Dark-Forest** [reply]
https://github.com/Hex-Dragon/PCL2/discussions/1807

**Reply by iPodtouch6557** [reply]
不信🤨，似乎是政治手段

**Reply by bmmNbhjd** [reply]
> 不信🤨，似乎是政治手段

确实是有的，但没多大用……

我要不挖坟那这人一辈子都不知道真相。

**Reply by Silverteal** [reply]
政治手段这个词可不是乱用的啊……勿挖坟
